frontend/themes/1/forum/views/forum/topic/posts/post/actions-post.tsx

```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import { MoreHorizontal, Pencil, Trash2 } from "lucide-react";
+import { useTranslations } from "next-intl";
+
+import { Button } from "@/components/ui/button";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuGroup,
+  DropdownMenuItem,
+  DropdownMenuTrigger
+} from "@/components/ui/dropdown-menu";
+import type { PermissionsTopicForums } from "@/graphql/hooks";
+import { useDeletePost } from "@/hooks/forum/posts/delete/use-delete-post";
+
+interface Props {
+  id: number;
+  permissions: PermissionsTopicForums;
+  state: {
+    locked: boolean;
+  };
+}
+
+export const ActionsPost = ({
+  id,
+  permissions: { can_edit },
+  state
+}: Props) => {
+  const t = useTranslations("forum.topics.actions");
+  const tCore = useTranslations("core");
+  const { deletePost } = useDeletePost({ id: id });
+  const editPost = async () => {
+    throw new Error("Not implemented yet");
```

Comment:
It breaks frontend on production. You can use `console.log` or simple `// TODO: something`